### PR TITLE
Add example of detectPointerOnOpaqueOnly

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

File: content/features/featuresDeepDive/gui/gui.md [hunk 1]
```diff
@@ -645,6 +647,8 @@ You can control the stretch used by the image with `image.stretch` property. You
 
 You may want to have the Image control adapt its size to the source image. To do so just call `image.autoScale = true`.
 
+### Image Source
+
```
Comment (sebavan) [inline content/features/featuresDeepDive/gui/gui.md:651]: not sure the extra line should be here ?

File: content/features/featuresDeepDive/gui/gui.md [hunk 2]
```diff
@@ -664,7 +668,11 @@ You can also apply stretch to animation sheet using `image.stretch` property.
 
 <Playground id="#K60448#2" title="Image With Stetched Sprite Sheet Example 2" description="Simple example showing how to add an Image with a stretched sprite sheet to your scene." image="/img/playgroundsAndNMEs/divingDeeperBabylonGUI36.jpg"/>
 
-Starting with babylon.js v4.0, you can also set `img.detectPointerOnOpaqueOnly = true` to indicate if pointers should only be validated on pixels with alpha > 0.
+### Pointer events
+
```
Comment (sebavan) [inline content/features/featuresDeepDive/gui/gui.md:672]: not sure the extra line should be here ?

**Review by RaananW — APPROVED** [review]
It is recommended to have an extra line after a markdown title (some parsers will fail or render incorrectly if it is not there)

**Comment by sebavan** [comment]
ahah I thought it was to be removed as I looked the first one on line 640 :-) so basically we should instead add one in 640

File: content/features/featuresDeepDive/gui/gui.md [hunk 3]
```diff
@@ -635,6 +635,8 @@ Here is an example of a MultiLine combining meshes, a control and a point: <Play
 ## Image
 
 Use the image control to display an image in your UI.
+
+### Stretch Modes and Scaling
 You can control the stretch used by the image with `image.stretch` property. You can set it to one of these values:
```
Comment (sebavan) [inline content/features/featuresDeepDive/gui/gui.md:640]: Missing extra line

**Pull request merged**